docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -24,54 +24,50 @@
 
 @endsphinxdirective
 
-## Introduction
-
 Model Optimizer is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
 
-Using Model Optimizer tool assumes you already have a deep learning model trained using one of the supported frameworks: TensorFlow, PyTorch, PaddlePaddle, MXNet, Caffe, Kaldi, or represented in ONNX* format. Model Optimizer produces an Intermediate Representation (IR) of the model, which can be inferred with [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
+When using Model Optimizer, you are assumed to already have a deep learning model that is trained using one of the supported frameworks: TensorFlow, PyTorch, PaddlePaddle, MXNet, Caffe, Kaldi, or represented in ONNX format. Model Optimizer produces an Intermediate Representation (IR) of the model, which can be inferred with [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
```

Comment:
```suggestion
To use it, you need a pre-trained deep learning model in one of the supported formats: TensorFlow, PyTorch, PaddlePaddle, MXNet, Caffe, Kaldi, or ONNX. Model Optimizer converts the model to the OpenVINO Intermediate Representation format (IR), which you can later infer with [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
```

Reply:
```suggestion
To use it, you need a pre-trained deep learning model in one of the supported formats: TensorFlow, PyTorch, PaddlePaddle, MXNet, Caffe, Kaldi, or ONNX. Model Optimizer converts the model to the OpenVINO Intermediate Representation format (IR), which you can infer later with [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
```
Tiny correction and it flies away :)

Reply:
I think there is a difference between "framework" and "format"? The original version says "trained using one of the supported frameworks: TensorFlow, PyTorch, PaddlePaddle, MXNet, Caffe, Kaldi, or represented in ONNX format", I would be careful to change the saying.

Reply:
The format results directly from the framework, so in this case it shouldn't be a problem. Even less so, if we have "onnx format" here. Besides, we always stress that it converts different model formats, so we can assume the same intention here ;)
And about the "later" thing, I wanted it to be in the meaning of "then" and not "in the future" :D but I won't pursue it. It has already flown away, so what would be the point ;)

Reply:
> 

I think we should remove the word "format" in this:
.....Model Optimizer converts the model to the OpenVINO Intermediate Representation **format** (IR), which you....

Reply:
I agree with linking for sure! Especially that we have this nice, long name here :)
But I am not convinced we should remove the word "format" OV IR is our model format. We should even reinforce the connection for our readers. We need to make people associate OpenVINO format with the IR format.

Reply:
I would also leave the word "format". The "IR format" phrase is quite popular throughout our documentation.